guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -359,7 +356,7 @@ stage:
 
 Set to `true` to skip static content deployment during the deploy phase.
 
-We recommend using this option, because running static content deployment during the deployment phase can greatly increase deployment times and downtime for your live site.
+We recommend setting this option to `true`, because running static content deployment during the deploy phase can greatly increase deployment times and downtime for your live site.
```

Comment:
Consider: significantly increase   

Comma before because is unneeded. See https://www.quickanddirtytips.com/education/grammar/when-to-use-a-comma-before-because.  (I've noticed this pattern across our doc set.)

`increase deployment times and downtime for your live site.` -- Nice description.    In the [Post-deploy phase](https://devdocs.magedevteam.com/hg-scdstrategies/guides/v2.2/cloud/deploy/cloud-deployment-process.html#-post-deploy-phase) description, it might be helpful to provide this detailed description in place of "dramatically affect site performance".